viewer/packages/sector-parser/visual-tests/GeneralCylinder.VisualTest.ts

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+import * as THREE from 'three';
+import {
+  createPrimitiveInterleavedGeometry,
+  GeneralCylinder,
+  PrimitiveName
+} from '../../../test-utilities/src/primitives';
+import { SimpleTestFixtureComponents, SimpleVisualTestFixture } from '../../../visual-tests';
+import { setConeGeometry } from '../src/reveal-glb-parser/primitiveGeometries';
+import { RevealGeometryCollectionType } from '../src/types';
+import { getMaterialsMap } from './testMaterials';
+
+export default class GeneralCylinderVisualTest extends SimpleVisualTestFixture {
```

Comment:
First of all, this test is just crazy useful and great - something that probably should be present for all primitives. Is it an idea to generate e.g. 10 different variations of the primitive to cover more possible regressions? I'm especially thinking edge cases